src/app/containers/Ad/Amp/__snapshots__/index.test.jsx.snap

```diff
@@ -46,54 +89,66 @@ exports[`AMP Ads Snapshots should correctly render an AMP leaderboard ad 1`] = `
         class="c0"
       >
         <div
-          amp-access="toggles.ads.enabled AND geoIp.advertiseCombined"
-          amp-access-hide="true"
+          class="c1"
         >
           <div
```

Comment:
@tochwill should we be applying aria hidden true somewhere here? Or is that covered in another piece of work?

Reply:
We will want to do this for both AMP and canonical, so there is a ticket here: https://github.com/bbc/simorgh/issues/6600 that covers that work.